vuepress/docs/next/docs/getting-started/development-concepts.md

```diff
@@ -0,0 +1,59 @@
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+
+	
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means developer teams can share components within a central catalog made available to multiple teams or organizations. Entando's distributed and composable development process allows you to quickly edit, test, analyze, and publish a singular component or packaged business capabilities (PBC) repeatedly, without touching other parts of your application. This includes the ability to publish content and pages along with functional components such as micro frontends and microservices. 
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
+| [Customize Bundle Info in App Builder](../curate/ecr-bundle-presentation-config.md) |   |
+| [Bundle Version and Updates FAQ](../curate/ecr-bundle-versions-faq.md) |  |
+| [Bundle Uninstall](../curate/ecr-uninstall-flow.md) |  |
+
+
+## Compose an Application 
+Business IT groups and developers Compose an Application with the Entando App Builder, a low-code composition UI. Assemble or update an application from the components already built in step one, adding templates, widgets, or full PBCs for functionality. The Entando Component Manager and Repository are integrated into the App Builder, making the composable process flexible and agile by managing all your components there. 
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Welcome Wizard](../compose/welcome-wizard.md) | [Page Management](../../tutorials/compose/page-management.md) |
+| [Entando Component Repository](../compose/ecr-overview.md) | [Widgets and Fragments](../../tutorials/compose/widgets-fragments.md) |
+| [Entando App Builder]<!--(../compose/app-builder.md)--> | [Content Creation](../../tutorials/compose/content-tutorial.md) |
+| [Entando Component Manager]<!--(../compose/ecm-overview.md)--> | [Content Templates](../../tutorials/compose/content-templates-tutorial.md)  |
+|  | [Digital Assets](../../tutorials/compose/digital-assets-tutorial.md) |
+
+## Consume Applications 
+In the Consume step, the end-user or consumer interacts with an application to complete specific tasks, the runtime phase of an Entando Application. The modular design of the platform allows the application and services to be scaled according to that use. In the same way, applications can be continuously improved by adding or updating web content, changing engine rules, and deploying seamless upgrades to individual components. 
```

Comment:
how is an end user different from a consumer? should be obvious the user is the one consuming. would remove hyphen and "consumer"

first sentence is awkward... not explicit if consumer interaction or task completion is the runtime phase; don't think mention of runtime phase adds value... nothing unique to Entando with respect to runtime and the concept of runtime isn't relevant here

sample rewording: "The end user Consumes content by interacting with an application to complete specific tasks. The modular design of the Entando Platform enables intelligent scaling of applications and services. This modularity also allows applications to be iteratively improved by adding or updating content, modifying engine operation, and deploying seamless upgrades to individual components."